monkey/monkey_island/cc/agent_event_handlers/update_nodes_on_exploitation.py

```diff
@@ -0,0 +1,14 @@
+from common.agent_events import ExploitationEvent
+from monkey_island.cc.models import CommunicationType
+from monkey_island.cc.repository import NetworkModelUpdateFacade
+
+
+class update_nodes_on_exploitation:
+    def __init__(self, network_model_update_facade: NetworkModelUpdateFacade):
+        self._network_model_update_facade = network_model_update_facade
+
+    def __call__(self, event: ExploitationEvent):
+        if event.success:
+            self._network_model_update_facade.upsert_communication_from_event(
```

Comment:
We should also update the machine with relevant services. It makes no sense that we used log4j to exploit tomcat on 8080, but still don't know what kind of service is running on 8080

Reply:
Agreed, but not now. Somewhere down the road.